src/dotnet/src/Mil.Navy.Nrl.Norm/NormData.cs

```diff
@@ -0,0 +1,39 @@
+﻿using System.Runtime.InteropServices;
+
+namespace Mil.Navy.Nrl.Norm
+{
+    /// <summary>
+    /// A transport object of type NORM_OBJECT_FILE.
+    /// </summary>
+    /// <remarks>
+    /// The data storage area for the specified transport object.
+    /// </remarks>
+    public class NormData : NormObject
+    {
+        /// <summary>
+        /// The data storage area associated with a transport object of type NORM_OBJECT_DATA.
+        /// </summary>
+        public byte[] Data
+        {
+            get
+            {
+                var dataPointer = NormDataAccessData(_handle);
+                var length = NormObjectGetSize(_handle);
+                var data = new byte[length];
+                for (var i = 0; i < length; i++)
+                {
+                    data[i] = Marshal.ReadByte(dataPointer, i);
```

Comment:
`Marshal.Copy` works better here.

Reply:
`Marshal.Copy` is a simpler approach than looping and reading each byte. Using a method would also better follow the C# Member Design Guidelines since it is a computationally complex operation. I will update to use `Marshal.Copy` and change from a property to a method.